bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/protocol/leap/LeapMessageParser.java

```diff
@@ -18,14 +18,7 @@
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
-import org.openhab.binding.lutron.internal.protocol.leap.dto.Area;
-import org.openhab.binding.lutron.internal.protocol.leap.dto.ButtonGroup;
-import org.openhab.binding.lutron.internal.protocol.leap.dto.Device;
-import org.openhab.binding.lutron.internal.protocol.leap.dto.ExceptionDetail;
-import org.openhab.binding.lutron.internal.protocol.leap.dto.Header;
-import org.openhab.binding.lutron.internal.protocol.leap.dto.OccupancyGroup;
-import org.openhab.binding.lutron.internal.protocol.leap.dto.OccupancyGroupStatus;
-import org.openhab.binding.lutron.internal.protocol.leap.dto.ZoneStatus;
+import org.openhab.binding.lutron.internal.protocol.leap.dto.*;
```

Comment:
Please revert as it introduces [checkstyle warnings](https://ci.openhab.org/job/PR-openHAB-Addons/17008/Static_20Code_20Analysis_20Report/)

Reply:
resolved